### DISCUSSION TITLE
Difference between top likelyhood-dynamical & velocity genes

### DISCUSSION THREAD

**Discussion by vim15** [theislab/scvelo#501] [Q&A]
I'm very sorry by the very basic question, but the more I read the info online and the subsequent paper the more I get confused. 

Running : 
`scv.tl.rank_velocity_genes 
`
or 

`scv.tl.rank_dynamical_genes`

will give you two very different gene lists per cluster that you have. My question is, what are these gene lists, what do they refer to in a biological context ???? 

In the tutorial for scVelo it says that these genes (velocity) are : "cluster-specific differential velocity expression, being siginificantly higher/lower compared to the remaining population."

and the dynamical genes are: "Driver genes display pronounced dynamic behavior and are systematically detected via their characterization by high likelihoods in the dynamic model."

But then wouldnt you expect that a driver gene would also be a high velocity gene as the cells would need more of it around to transition to a different lineage? 

If you could give a simple explanation (please more biological if possible than mathematical)as to what these different sets of genes refer to, and how this might be applied to biology it would be greatly appreciated.

**Comment by andrea-de-micheli** [comment]
Hi!
I'm wondering the same. Anyone has a simple biological explanation? 
The gene lists I get from `scv.tl.rank_velocity_genes` and `scv.tl.rank_dynamical_genes` are also rather different in my case.
Thank you!

**Comment by DaianeH** [comment]
Wondering the same. My lists from scv.tl.rank_velocity_genes and scv.tl.rank_dynamical_genes are completely different.

**Comment by ishwarvh** [comment]
This needs better clarification, it almost appears like they are being used interchangeably throughout the documentation. The best approximate distinction I could come up with after scouring through the tutorial and the documentation is:

`Velocity genes` : are computed based on the splicing kinetics of genes in all the cells and are used to model the cell fate transition probabilities, which is what is used to project the arrows.

`Dynamic genes`: are cluster-specific genes with a higher unspliced-to-splice ratio.

Will be nice to have @VolkerBergen provide some clarification though. Thank you so much for opening this whole new world of analysis with your package.
